framework/files/system/oem/99_elemental_system_agent.yaml

```diff
@@ -0,0 +1,7 @@
+name: "Elemental system agent bootstarp"
+stages:
+  network.after:
+    # run only if on live cd and there is a config file
+    - if: '[ ! -f /run/cos/live_mode ] && [ ! -f /etc/systemd/system/rancher-system-agent.service ]'
```

Comment:
what about recovery? shouldnt we just launch this under active partition?

Reply:
good point